src/Model/Entity/Panel.php

```diff
@@ -46,10 +46,37 @@ class Panel extends Entity
      */
     protected function _getContent(mixed $content): string
     {
+        if (is_string($content)) {
+            $contentInflated = @gzinflate($content);
+            if ($contentInflated) {
```

Comment:
```suggestion
            if ($contentInflated !== false) {
```

comparing to false will avoid empty string from being ignored.